src/app/sidebar/components/CardDropDown.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import { Box, Collapse, Flex, Heading, IconButton } from "@chakra-ui/react";
+import styled from "@emotion/styled";
+import { ChevronDownIcon } from "@chakra-ui/icons";
+import { Colors } from "../shared/styles";
+
+export interface CardProps {
+    /*
+    * The subject
+    * Ex/ CSC
+    */
+    subject: string;
+    /*
+    * The subject title
+    * Ex/ Computer Science
+    */
+    title: string;
+    /*
+    * The list of courses for this subject
+    */
+    courseCards: JSX.Element[];
+}
+
+export function CardDropDown(props: CardProps): JSX.Element {
+    const [isDisplayed, setDisplayed] = useState(false);
+
+    const handleClick = () => {
+        setDisplayed(!isDisplayed)
+    }
+
+    return (
+        <Flex direction="column">
+            <StyledBox boxShadow="lg">
+                <Flex direction="row" alignItems="center" justifyContent="space-between">
+                    <Heading size="sm" color={Colors.black} fontWeight={"normal"}>
+                        {props.subject} - {props.title}
```

Comment:
Not in the Figma but I feel the title and subject should have a bit more contrast between them even if it's subtle.

Reply:
<img width="289" alt="Screen Shot 2021-02-28 at 5 09 15 PM" src="https://user-images.githubusercontent.com/31324580/109440607-b837c100-79e7-11eb-9524-37c601caafc1.png">